### ISSUE TITLE
any way to restart counter?

### ISSUE THREAD

**Issue by defji** [fengyuanchen/vue-countdown#39]
No method for restarting counter.

**Comment by fengyuanchen** [comment]
Just set `time` prop to its initial value again.

**Comment by fengyuanchen** [comment]
Or use `v-if` to rerender the component.

**Comment by defji** [comment]
thanks dude! 🤘

**Comment by simonschaufi** [comment]
@fengyuanchen @defji I linked a new issue with this as I have the problem that setting the prop works only once. The second time the counter just continues to count down. Can anyone reproduce this?

**Comment by craigh411** [comment]
For anybody else looking at this, the simplest way to reset the timer is to set the time prop value to 0 and then reset it back using $nextTick:

```
this.time = 0
this.$nextTick(() => {
   this.time = 30000
})
```
